### ISSUE TITLE
Frame with both optional and overlay specification can give frame-break

### ISSUE THREAD

**Issue by blefloch** [josephwright/beamer#743]
A frame can have both an overlay specification in angle brackets and an optional argument.  In that order, it works (see frame-A below).  But in the other order, one gets `<1>` typeset, with `<` alone on one frame, and `1>` on the next one, which is pretty puzzling.
```
\documentclass{beamer}
\begin{document}
\begin{frame}<1>[label=frame-A]
\end{frame}
\begin{frame}[label=frame-B]<1>
\end{frame}
\end{document}
```

**Comment by samcarter** [comment]
Only the first version is valid syntax, the frame environment is documented like this:

```
\begin{frame}<⟨overlay specification⟩>[<⟨default overlay specification⟩>][⟨options⟩]{⟨title⟩}{⟨subtitle⟩} 
⟨environment contents⟩
\end{frame}
```

**Comment by blefloch** [comment]
Thanks.  Nevertheless the second one should not give two frames but a single frame with `<1>` being typeset.

**Comment by samcarter** [comment]
Even in the middle of a frame, a `<1>` won't be typeset as "<1>" (at least not without additional tricks)
![document](https://user-images.githubusercontent.com/43832342/145021509-238c3a51-9e6d-41a4-8980-b897309fe8b3.png)

**Comment by blefloch** [comment]
Correct, I should have added `\usepackage[T1]{fontenc}` to preempt this point.  My surprise is with the fact that there is a frame break: `<` (or rather `¡`) gets on one page and `1>` (or rather `1¿`) on the next.
